netutils/esp8266/esp8266.c

```diff
@@ -2672,8 +2673,10 @@ int lesp_setsockopt(int sockfd, int level, int option,
           case SO_RCVTIMEO:
               if (value_len == sizeof(struct timeval))
                 {
-                  sock->rcv_timeo.tv_sec = ((struct timeval *)(value))->tv_sec;
-                  sock->rcv_timeo.tv_nsec = ((struct timeval *)(value))->tv_usec;
+                  sock->rcv_timeo.tv_sec = ((struct timeval *)
+                                            (value))->tv_sec;
+                  sock->rcv_timeo.tv_nsec = ((struct timeval *)
+                                             (value))->tv_usec;
```

Comment:
Maybe next will satisfy the style
```suggestion
                  sock->rcv_timeo.tv_sec = ((struct timeval *)value)->tv_sec;
                  sock->rcv_timeo.tv_nsec = ((struct timeval *)value)->tv_usec;
```

Reply:
there is no harm to keep the old code. we can refactor in another commit